src/main/java/org/apache/sling/distribution/event/DistributionEventProperties.java

```diff
@@ -47,4 +47,9 @@
      * property containing the type of the distribution paths
      */
     String DISTRIBUTION_PATHS= "distribution.paths";
+    
+    /**
+     * Package id
+     */
+    String PACKAGE_ID = "distribution.package.id";
```

Comment:
How about naming the constant `DISTRIBUTION_PACKAGE_ID` to be consistent with the other properties ?